atlite/data.py

```diff
@@ -21,16 +24,20 @@ def get_features(cutout, module, features, tmpdir=None):
     Load the feature data for a given module.
 
     This get the data for a set of features from a module. All modules in
-    `atlite.datasets` all allowed.
+    `atlite.datasets` are allowed.
     """
     parameters = cutout.data.attrs
+    lock = SerializableLock()
     datasets = []
     get_data = datamodules[module].get_data
 
     for feature in features:
-        feature_data = get_data(cutout, feature, tmpdir=tmpdir, **parameters)
+        feature_data = get_data(cutout, feature, tmpdir=tmpdir, lock=lock, **parameters)
         datasets.append(feature_data)
```

Comment:
The expected output of all `get_data` functions should be the same, always a `xr.Dataset`, not delayed objects. What speaks against adding the delay here? (instead of within `era5.py`)
`delayed(get_data)(cutout, feature, tmpdir=tmpdir, lock=lock, **parameters)`
Then we'd also have a parallel queuing

Reply:
Not for multiple years in the same dataset. Maybe that's not soo much of a concern, since the standard is a single download per feature, i guess.